### ISSUE TITLE
Unable to Import Records Into Algolia

### ISSUE THREAD

**Issue by foodlyhenry** [algolia/firestore-algolia-search#46]
Hi, I have a very similar issue to:
[https://github.com/algolia/firestore-algolia-search/issues/43#event-4954001620](url)
and
[https://github.com/algolia/firestore-algolia-search/issues/42#issuecomment-869508635](url)

Ignoring the error messages like for issue #42 and using the one liner command provided in #43 did not solve my problem.

Here is the command I ran:
`LOCATION=us-central1 ALGOLIA_APP_ID=D********* ALGOLIA_API_KEY=b****************0 ALGOLIA_INDEX_NAME=dev_foodly COLLECTION_PATH=Restaurants/{restaurantId}/Locations FIELDS=restaurantName GOOGLE_APPLICATION_CREDENTIALS=foodlyServiceAccountKeyDev.json npx firestore-algolia-search@0.4.1
`
And this is my output:
```
{"severity":"WARNING","message":"Warning, FIREBASE_CONFIG and GCLOUD_PROJECT environment variables are missing. Initializing firebase-admin will fail"}
{"algoliaAppId":"D7MMB2JA20","algoliaAPIKey":"********","algoliaIndexName":"dev_foodly","collectionPath":"Restaurants/{restaurantId}/Locations","fields":"restaurantName","severity":"INFO","message":"Initializing extension with configuration"}

WARNING: The back fill process will index your entire collection which will impact your Search Operation Quota.  Please visit https://www.algolia.com/doc/faq/accounts-billing/how-algolia-count-records-and-operation/ for more details.  Do you want to continue? (y/N): y
✨  Done in 41.50s.
```

**Comment by smomin** [comment]
I am not seeing any error message but warnings.  Can you send me a screen shot of your API permissions?  (make sure to black out sensitive data)

**Comment by foodlyhenry** [comment]
Here it is, sorry for the late reply. Is this what you needed?
![Screen Shot 2021-07-08 at 3 00 23 PM](https://user-images.githubusercontent.com/79164619/124996095-3abb3780-dffd-11eb-8077-18fe72a81c2a.png)

**Comment by smomin** [comment]
I am curious if your collection path is incorrect.  Can you try changing it to `Restaurants`?

**Comment by foodlyhenry** [comment]
Changing to Restaurants does import the records to Algolia, However what I want is to import the Locations (Restaurants/{restaurantId}/Locations) into Algolia to search. 

Also, I forgot to mention that Algolia is properly importing records into the index, however this only happens when I create a new Restaurant not when running the command. 

For example If i created a new Document inside 'Restaurants/{restaurantId}/Locations', then it is imported into Algolia automatically as expected.

**Comment by smomin** [comment]
understood.  the issue is on the import script and not the real time indexing.  will take a look.

**Comment by smomin** [comment]
This code was merged with the new version 0.5.0.
